mne/surface.py

```diff
@@ -1666,23 +1667,47 @@ def marching_cubes(image, level):
     ----------
     image : ndarray
         The image to compute marching cubes with.
-    level : float
+    level : float | int | list
         The contour value to search for isosurfaces in ``image``.
+        Can also be a list of integers to use a discrete marching cubes
+        algorithm suitable for labeled volumes (e.g., ``'aseg.mgz'``).
+    %(smooth)s Must be between 0 (inclusive; default) and 1.
```

Comment:
This extra text should go in `docs.py` not here (it's the same in both cases)

Reply:
Great, will do